services/src/main/java/org/keycloak/organization/admin/resource/OrganizationMemberResource.java

```diff
@@ -97,6 +99,10 @@ public Response addMember(String id) {
 
         try {
             if (provider.addMember(organization, user)) {
+                adminEvent.operation(OperationType.CREATE).resource(ResourceType.ORGANIZATION_MEMBERSHIP)
```

Comment:
would be nice to also add the ID of the member to the event. Same for remove. See https://github.com/keycloak/keycloak/issues/34013

https://github.com/keycloak/keycloak/blob/main/services/src/main/java/org/keycloak/organization/admin/resource/OrganizationMemberResource.java

Reply:
@Degoya-Studio - please create a new issue for this and reference it here.

Reply:
... or update the issue you referenced accordingly.